nmostesting/suites/IS0401Test.py

```diff
@@ -1351,7 +1351,12 @@ def test_20_01(self, test):
                     access_error = True
                     continue
 
-                valid, response = self.do_request("GET", href)
+                if self.authorization and href is not None and urlparse(href).path.startswith("/x-nmos/connection"):

```

Comment:
I think `href is not None` can be removed here because of the `if not href` just above?